### DISCUSSION TITLE
BLM banner irrelevance to the RxJS - platform misuse

### DISCUSSION THREAD

**Discussion by RAYDENFilipp** [ReactiveX/rxjs#6197] [Ideas / Feature request]
I have seen there were several issues being opened and closed regarding the rationale of the BLM banner on the GitHub page of the RxJS project some of which were closed with fruitless discussions that were rather opinionated regarding the BLM itself and how it does not relate to the experiences of developers worldwide than platform misuse.

  My point is not to discuss the BLM itself and how it is irrelevant to issues a library user is facing on a daily basis in their country of residence (which are plenty, most African countries with the shortage of water and food, people in the Middle East suffered the atrocities of the ISIS, ongoing war in Ukraine, military coup d'etat in Myanmar, Uyghurs being repressed in China. HK protests and many more - thousands of them and each are most important to those, who live in these countries, while for others is just a newsfeed on a newsweekly), but none of these issues are not reflected on any banner of any libraries I have ever seen (please, correct me if I am wrong, but I didn't see any). Not even COVID-19, that I can remember (but maybe I didn't notice).

  Nobody wants to say these issues are less important than those that happened in the US, I hope. But none reflected.

  There is no need to mention that the donation links are for,
"
to end mass incarceration and excessive punishment in the United States, challenge racial and economic injustice, and protect the basic human rights for the most vulnerable people in American society
" - ​Equal Justice Initiative

so, again, not a single community worldwide will benefit from it.

  What I suggest:

1. Keeping the RxJS GitHub-page clean of any social activities and as professionally focused on its domain as possible. 
It is ok to promote social initiatives and there are plenty of mediums for that: Facebook, Twitter, etc.

1. Or, updating the banner for every issue that happens around the world and promote donations that are of importance to the user's location. But that will be an overhead to the project and a matter of irrelevance, again.

P.S. this thread is not to discuss the BLM, nor any other issue in the world. It is simply a matter of platform misuse.

**Comment by jorroll** [comment]
Following the OP's lead and putting aside BLM itself, IMO the volunteer maintainers of RXJS ([mostly 6 ppl in the last year](https://github.com/ReactiveX/rxjs/graphs/contributors?from=2019-12-31&to=2021-04-17&type=c)) are doing a great service to all of us by freely donating their time to building and maintaining this open source repository. If they want to use a platform which they've built over hundreds of hours to advocate for issues that are important to them, the absolute least I can do is listen.

> Nobody wants to say these issues are less important than those that happened in the US, I hope. But none reflected.

Why is it unreasonable for the maintainers to promote the issues that are most important to them? After all, they are the ones providing a free service and it's their platform. If they were promoting a message that was in any way offensive, I could understand complaining about it. But if your main argument is "I don't feel like BLM issues apply to me," I'd say the polite thing to do is just click past the messaging and be grateful that the only thing the maintainers ask for in return for their hundred's of hours of volunteer service is that you consider the BLM movement when browsing this repo's documentation. It feels pretty selfish to take advantage of all this free labor and then complain when they add a personal message to the documentation.

> none of these issues are not reflected on any banner of any libraries I have ever seen

FTR, many open source repos add personal messages to their documentation. I think what I see most frequently is a message that the author is looking for a job or a request for donations or an advertisement from a sponsor, but I've seen numerous library's voice messages of support for BLM. For example, [angular.io](https://github.com/angular/angular/issues/37771) used to have a message of support for BLM (I linked to an old issue complaining about the previous messaging as proof) and the [`core-js`](https://github.com/zloirock/core-js) readme still includes an advertisement from the maintainer about looking for a job.

**Reply by RAYDENFilipp** [reply]
Well, if put this way that maintainers are free to do what they want as long as they don't charge others for the use of their work then yes, I see the point. Makes sense now. Thanks for pointing it out from this perspective.
